hazelcast/src/test/java/com/hazelcast/Main.java

```diff
@@ -0,0 +1,55 @@
+package com.hazelcast;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.internal.diagnostics.Diagnostics;
+import com.hazelcast.nio.tcp.TcpIpConnection;
+import com.hazelcast.test.HazelcastTestSupport;
+
+/**
+ * Created by alarmnummer on 5/2/16.
+ */
+public class Main extends HazelcastTestSupport {
```

Comment:
I guess this class does not have to be merged; its tests could be integrated in `SystemLogPluginTest`.